clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/agents/TitusStreamingUpdateAgent.java

```diff
@@ -316,8 +315,11 @@ private void writeToCache(StreamingCacheState state) {
       long startTime = System.currentTimeMillis();
 
       if (!state.savedSnapshot ||
-        state.changes.get() >= ITEMS_CHANGED_THRESHOLD ||
-        (startTime - state.lastUpdate.get() > TIME_UPDATE_THRESHOLD_MS && state.changes.get() > 0)
+        state.changes.get() >=
+          dynamicConfigService.getConfig(Integer.class, "titus.streaming.changeThreshold", 1000) ||
```

Comment:
Assuming the change in default values was conscious?

Reply:
Yep, good question